qualtran/bloqs/arithmetic/bitwise.py

```diff
@@ -50,33 +47,23 @@ def _cvs_converter(vv):
 
 
 @frozen
-class XorK(SpecializedSingleQubitControlledGate):
+class XorK(Bloq):
```

Comment:
While we are at it, let's also override the `wire_symbol` method for both the bloqs in this file and return `TextBox(f"⊕ {K}")` for the `XorK` and `TextBox(f"x⊕y")` for `Xor` for the target registers

Reply:
added wire symbols and a circuit diagram unit test